Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,15 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+
+Excercise 1:
+
+1- Since we have n * n * n inside the while loop, this would be O(n^3)
```

Comment:
Notice the problem is a little bit tricky.
While the loop condition goes up to n^3, notice that inside the loop each increment is by n^2,